client/src/components/classes/SenderForm.js

```diff
@@ -1,18 +1,124 @@
-import React from "react";
+import React, { useState } from "react";
 import Grid from "@material-ui/core/Grid";
 import { withStyles } from "@material-ui/core/styles";
 
 const styles = theme => ({
   wrapper: {
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    width: "25%",
+    '& input': {
+      margin: "5px",
+    }
   }
 });
 
 function SenderForm(props) {
+  const [nickname, setNickname] = useState("")
+  const [senderName, setSenderName] = useState("")
+  const [senderEmail, setSenderEmail] = useState("")
+  const [replyName, setReplyName] = useState("")
+  const [replyEmail, setReplyEmail] = useState("")
+  const [address, setAddress] = useState("")
+  const [address2, setAddress2] = useState("")
+  const [state, setState] = useState("")
+  const [zip, setZip] = useState("")
+  const [city, setCity] = useState("")
+  const [country, setCountry] = useState("")
+
```

Comment:
Yay hooks!

One other way I found to do this, similar to using dynamic handleChange in classes, was to set all of these fields an object. Then you can use the same handleChange for any update, ie
```
const BillingPage = props => {
￼	  const [cardInfo, setCardInfo] = useState({
￼	    cardNum: '',
￼	    expiration: '',
￼	    cvv: '',
￼	    subType: '',
￼	  });
￼	
￼	  const handleChange = e => {
￼	    setCardInfo({ ...cardInfo, [e.target.name]: e.target.value });
```

Reply:
Wow, that's neat! We can talk more about that structure today too.

I just slapped something together to get the concept flushed out!